ft-email-auth-provider/src/handlers/confirm_email.rs

```diff
@@ -0,0 +1,92 @@
+use crate::handlers::resend_confirmation_email::{
+    generate_new_confirmation_key, send_confirmation_email,
+};
+use validator::ValidateEmail;
+
+#[ft_sdk::form]
+pub fn confirm_email(
+    mut conn: ft_sdk::Connection,
+    ft_sdk::Query(code): ft_sdk::Query<"code">,
+    ft_sdk::Query(email): ft_sdk::Query<"email">,
+    host: ft_sdk::Host,
+    mountpoint: ft_sdk::Mountpoint,
+) -> ft_sdk::form::Result {
+    if !email.validate_email() {
+        return Err(ft_sdk::single_error("email", "invalid email format").into());
+    }
+
+    let (user_id, data) = ft_sdk::auth::provider::user_data_by_custom_attribute(
+        &mut conn,
+        crate::PROVIDER_ID,
+        crate::EMAIL_CONF_CODE_KEY,
```

Comment:
Don't use `crate`, use `auth`.

Reply:
done in https://github.com/fastn-community/auth/pull/4/commits/e90c88048b1497f6cf86b0024118f1e49cd31768